### PR TITLE
sql: update the transaction related documents

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

As the title says.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: sql/tidb-specific.md [hunk 1]
```diff
@@ -313,18 +313,30 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 ### tidb_disable_txn_auto_retry
 
 作用域：SESSION | GLOBAL
 
-默认值：0
+默认值：1
 
 这个变量用来设置是否禁用显式事务自动重试，设置为 1 时，不会自动重试，如果遇到事务冲突需要在应用层重试。
 
+这个变量不会影响自动提交的隐式事务和 TiDB 内部执行的事务，它们依旧会根据 tidb_retry_limit 的值来决定最大重试次数。
```
Comment (lilin90) [inline sql/tidb-specific.md]:
```suggestion
这个变量不会影响自动提交的隐式事务和 TiDB 内部执行的事务，它们依旧会根据 `tidb_retry_limit` 的值来决定最大重试次数。
```

File: sql/tidb-specific.md [hunk 2]
```diff
@@ -313,18 +313,30 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
+
 是否需要禁用自动重试，请参考[自动重试的风险](/sql/transaction-isolation.md#乐观事务注意事项)。
 
+### tidb_back_off_weight
+
+作用域：SESSION | GLOBAL
+
+默认值：2
+
+这个变量用来给 TiDB 的 back-off 最大时间增加权重，即内部遇到网络或其他组件（TiKV、PD）故障等时，发送重试请求的最大重试时间，可以通过这个变量来调整。最小值为 1。
```
Comment (lilin90) [inline sql/tidb-specific.md]:
```suggestion
这个变量用来给 TiDB 的 `back-off` 最大时间增加权重，即内部遇到网络或其他组件（TiKV、PD）故障等时，发送重试请求的最大重试时间。可以通过这个变量来调整最大重试时间，最小值为 1。
```

File: sql/tidb-specific.md [hunk 3]
```diff
@@ -313,18 +313,30 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
+如，TiDB 向 PD 取 tso 的基础超时时间是 15 秒，当 tidb_back_off_weight = 2 时，取 tso 的最大超时时间为基础时间 * 2 等于 30 秒。
```
Comment (lilin90) [inline sql/tidb-specific.md]:
```suggestion

例如，TiDB 向 PD 取 TSO 的基础超时时间是 15 秒，当 `tidb_back_off_weight = 2` 时，取 TSO 的最大超时时间为：基础时间 * 2 等于 30 秒。
```

File: sql/tidb-specific.md [hunk 4]
```diff
@@ -313,18 +313,30 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
+当网络环境较差的情况下，适当增大这个变量值，可以有效缓解因为超时而向应用端报错的情况；而如果应用端希望更快的接到报错信息，则应该尽量减小这个变量的值。
```
Comment (lilin90) [inline sql/tidb-specific.md]:
```suggestion

在网络环境较差的情况下，适当增大该变量值可以有效缓解因为超时而向应用端报错的情况；而如果应用端希望更快地接到报错信息，则应该尽量减小该变量的值。
```

File: sql/transaction-isolation.md [hunk 5]
```diff
@@ -7,20 +7,20 @@ category: user guide
 
 事务隔离级别是数据库事务处理的基础，ACID 中 I，即 Isolation，指的就是事务的隔离性。
 
-sql 92标准定义了4种隔离级别，读未提交、读已提交、可重复读、串行化，见下表。
+SQL 92标准定义了4种隔离级别，读未提交、读已提交、可重复读、串行化，见下表。
```
Comment (lilin90) [inline sql/transaction-isolation.md]:
```suggestion
SQL 92 标准定义了 4 种隔离级别：读未提交、读已提交、可重复读、串行化。详见下表：
```

File: sql/transaction-isolation.md [hunk 6]
```diff
@@ -7,20 +7,20 @@ category: user guide
...
 
-| Isolation Level  | Dirty Read   | Nonrepeatable Read | Phantom Read          | Serialization Anomaly |
-| ---------------- | ------------ | ------------------ | --------------------- | --------------------- |
-| Read uncommitted | Possible     | Possible           | Possible              | Possible              |
-| Read committed   | Not possible | Possible           | Possible              | Possible              |
-| Repeatable read  | Not possible | Not possible       | Not possible in  TiDB | Possible              |
-| Serializable     | Not possible | Not possible       | Not possible          | Not possible          |
+| Isolation Level  | Dirty Write  | Dirty Read   | Fuzzy Read   | Phantom      |
```
Comment (lilin90) [inline sql/transaction-isolation.md]:
```suggestion
| Isolation Level  | Dirty Write  | Dirty Read   | Fuzzy Read   | Phantom Read |
```

File: sql/transaction-model.md [hunk 7]
```diff
@@ -58,7 +58,7 @@ COMMIT;
 
     其中 ESCAPED BY 目前只支持 '/\/\'。
 
-+   事务的处理：
-
-    TiDB 在执行 load data 时，默认每 2 万行记录作为一个事务进行持久化存储。如果一次 load data 操作插入的数据超过 2 万行，那么会分为多个事务进行提交。如果某个事务出错，这个事务会提交失败，但它前面的事务仍然会提交成功，在这种情况下一次 load data 操作会有部分数据插入成功，部分数据插入失败。而 MySQL 中会将一次 load data 操作视为一个事务，如果其中发生失败情况，将会导致整个 load data 操作失败。
+* 事务的处理：
 
+    TiDB 在执行 LOAD DATA 时，默认每 2 万行记录作为一个事务进行持久化存储。如果一次 LOAD DATA 操作插入的数据超过 2 万行，那么会分为多个事务进行提交。如果某个事务出错，这个事务会提交失败，但它前面的事务仍然会提交成功，在这种情况下一次 LOAD DATA 操作会有部分数据插入成功，部分数据插入失败。而 MySQL 中会将一次 LOAD DATA 操作视为一个事务，如果其中发生失败情况，将会导致整个 LOAD DATA 操作失败。
```
Comment (lilin90) [inline sql/transaction-model.md]:
```suggestion
    TiDB 在执行 `LOAD DATA` 操作时，默认将每 2 万行记录作为一个事务进行持久化存储。如果一次 `LOAD DATA` 操作插入的数据超过 2 万行，那么会分为多个事务进行提交。如果某个事务出错，这个事务会提交失败，但它前面的事务仍然会提交成功，在这种情况下，一次 `LOAD DATA` 操作会有部分数据插入成功，部分数据插入失败。而 MySQL 中会将一次 `LOAD DATA` 操作视为一个事务，如果其中发生失败情况，将会导致整个 `LOAD DATA` 操作失败。
```

File: sql/transaction-model.md [hunk 8]
```diff
@@ -58,7 +58,7 @@ COMMIT;
...
+    **即，LOAD DATA 在 TiDB 中没有事务原子性的保证，不建议在生产环境中使用。**
```
Comment (lilin90) [inline sql/transaction-model.md]:
```suggestion
    > **注意：**
    >
    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，不建议在生产环境中使用。
```

File: sql/transaction.md [hunk 9]
```diff
@@ -62,13 +64,13 @@ ROLLBACK;
 
 TiDB 可以显式地使用事务（`BEGIN/COMMIT`）或者隐式的使用事务（`SET autocommit = 1`）。
 
-如果在 autocmmit = 1 的状态下，通过 `BEGIN` 语句开启一个新的事务，那么在 `COMMIT`/`ROLLBACK` 之前，会禁用 autocommit，也就是变成显式事务。
+如果在 autocommit = 1 的状态下，通过 `BEGIN` 语句开启一个新的事务，那么在 `COMMIT`/`ROLLBACK` 之前，会禁用 autocommit，也就是变成显式事务。
```
Comment (lilin90) [inline sql/transaction.md]:
```suggestion
如果在 `autocommit = 1` 的状态下，通过 `BEGIN` 语句开启一个新的事务，那么在 `COMMIT`/`ROLLBACK` 之前，会禁用 autocommit，也就是变成显式事务。
```

File: sql/transaction-isolation.md (hunk 6 above)
Comment (jackysp) [reply sql/transaction-isolation.md]: It should be no `Read` here.

Comment (lilin90) [reply sql/transaction-isolation.md]: Got it.

**Pull request merged**